monitoring/README.md

```diff
@@ -1,4 +1,4 @@
-
+# Audius Network Monitoring
```

Comment:
monitoring repo is different from "Network Monitoring", which is a discovery plugin, living in `discovery-provider/plugins/network-monitoring`

this dir is intended to house general monitoring tools - may not be the best place, we just picked a place for now

Reply:
cc @jonaylor89

Reply:
Yup, Network Monitoring is coming to the protocol repo soon. This `monitoring/` directory is just for general tools and services used in our monitoring stack

Reply:
To suggest something quick, maybe 

# Monitoring Stack

is a more appropriate header?

Reply:
`monitoring-tools`?

Reply:
i opened a Pr against this one, if you guys think its cool can merge into here
https://github.com/AudiusProject/audius-protocol/pull/3229

Reply:
Hey! I super appreciate you opening up a PR against this one! 🤗 

I don't see `discovery-provider/plugins/network-monitoring` in `master`. What branch is that coming in from?

I would argue that a) this directory is closer to that of `monitoring-stack` than `monitoring-tools` since these aren't a collection of scripts so much as they are a collection of long running softwares. If you wouldn't mind, what are your thoughts on `monitoring-stack`?

And b) we should rename `discovery-provider/plugins/network-monitoring` to be `discovery-provider/plugins/content-network-monitoring` since the Network Monitoring Redesign is focused on the replication of content/data across our network of Content Nodes where this directory not only displays those metrics, but metrics from all other parts of the system (GCP, AWS, Pingdom, our ELK logging configuration files would probably go well here, and I would argue community metrics (Twitter, Discord, Telegram, Reddit, Instagram, etc) would be a great candidates to track as a way to help predict/diagnose user activity). Given that this directory is a super set of all monitoring, I'd be more specific of what `discovery-provider/plugins/*-network-monitoring` was named to avoid confusion in the future. [Note that this conversation doesn't need to take place now, just considered for future work.]

If `monitoring` is still not ideal, what are your thoughts on renaming to `monitoring-stack` instead?

Reply:
fair point joaquin - reverted to `monitoring` since that name is simplest. i dont love monitoring-stack

`discovery-provider/plugins/network-monitoring` doesn't exist yet, @jonaylor89 is going to set that up soon

the purpose of NetworkMonitoring is not jsut for content monitoring, but to be a plugin for all network metrics, derivable from a discovery DB + content dataset

agreed with you, lets discuss this separately